contrib/mpi-proxy-split/lower-half/gethostbyname-static/Makefile

```diff
@@ -24,7 +24,10 @@ dist: clean
 	dir=`basename $$PWD` && cd .. && tar zcvf ./$$dir.tar.gz ./$$dir
 	dir=`basename $$PWD` && ls -l ../$$dir.tar.gz
 
-install:
+mana:
 	make gethostbyname_proxy
-	cp gethostbyname_proxy -f ../../../../bin
+	install gethostbyname_proxy ../../../../bin
```

Comment:
would `gethostbyname_proxy` be copied to the `DESTDIR`?

Reply:
yes, but it's done in the parent directory's Makefile. i'm trying to see if i move all the making of `gethostbyname-static` or `gethostbyname-proxy` to this Makefile.